### PR TITLE
Add UML for profile

### PR DESCRIPTION
**Pull request by calebschoepp**
Super duper simple UML for Profile. Looking for commentary on how in depth I should go and also if I'm following the pattern correctly.

Screenshot from 2020-10-10 21-08-41

### CODE REVIEW

**Comment by ghassanzadeh** [comment]
It looks good with what we have so far. I don't think we need to add too much details for the first project deliverable.

**Review by ghassanzadeh — APPROVED** [review]

**Review by cyrusdiego — COMMENTED** [review]
I think it looks good to me as well, though I'm wondering if we should be adding comments to explain the MVVM architecture, thoughts on this?

Also in terms of detail, this may be worth asking the TA or prof or looking at the forum, but did they mention if we're supposed to have like all the different views in the UML (i.e. EditText, Buttons, etc.). Ideally, for us, it would just be the general architecture and we don't need to include all the little UI components in this.

**Comment by calebschoepp** [comment]
> I think it looks good to me as well, though I'm wondering if we should be adding comments to explain the MVVM architecture, thoughts on this?

IMO if we do add comments regarding the MVVM architecture it should just be an overall comment referring to everything. Not on each set.

> Also in terms of detail, this may be worth asking the TA or prof or looking at the forum, but did they mention if we're supposed to have like all the different views in the UML (i.e. EditText, Buttons, etc.). Ideally, for us, it would just be the general architecture and we don't need to include all the little UI components in this.

IMO they probably don't want the specificity of individual view components for this first UML. We don't even really have any code to base it off of yet.

**Comment by cyrusdiego** [comment]
> IMO if we do add comments regarding the MVVM architecture it should just be an overall comment referring to everything. Not on each set.
Yeah, I think an overall comment would be good. Something to add during our next meeting most likely. 

> IMO they probably don't want the specificity of individual view components for this first UML. We don't even really have any code to base it off of yet.
Yeah, that's what I figured as well. 

Besides those questions/comments I think the UML looks good!

File: doc/p1-uml.txt [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-  void methods()
+class ProfileActivity {
+  #void onCreate()
 }
 
-class Flight {
-   flightNumber : Integer
-   departureTime : Date
+class ProfileViewModel {
+  +LibraryViewModel()
```
[the remark this answers is not in the document]
Comment (calebschoepp) [reply doc/p1-uml.txt:7]: Oh yeah it totally is supposed to be `ProfileViewModel()`

Comment (calebschoepp) [inline doc/p1-uml.txt:7]:
```suggestion
  +ProfileViewModel()
```

[the remark this answers is not in the document]
Comment (armiantos) [reply doc/p1-uml.txt]: Sweet!

**Review by armiantos — APPROVED** [review]
LGTM

**Pull request merged**